scripts/buck_setup.sh

```diff
@@ -1,17 +1,31 @@
 #!/bin/bash
 printf "\nCreating .buckconfig\n"
```

Comment:
Would be much better if this used `set -e`, so errors were obvious

Reply:
Agreed